Header header
sensor_msgs/Image image
sensor_msgs/CameraInfo camInfo